src/batch-submitter/tx-batch-submitter.ts

```diff
@@ -187,17 +201,29 @@ export class TransactionBatchSubmitter extends BatchSubmitter {
         }
       }
     }
+
+    this.log.debug('Set lastL1BlockNumber', {
+      lastL1BlockNumber: this.lastL1BlockNumber,
+    })
   }
 
   public async _getBatchStartAndEnd(): Promise<Range> {
     // TODO: Remove BLOCK_OFFSET by adding a tx to Geth's genesis
     const startBlock =
       (await this.chainContract.getTotalElements()).toNumber() + BLOCK_OFFSET
+    this.log.info('Retrieved start block number from CTC', {
+      startBlock,
+    })
```

Comment:
Might be useful to log both before and after this asynchronous network call for debugging purposes